### ISSUE TITLE
streetview: sample in readme doesn't work

### ISSUE THREAD

**Issue by lawm** [googlemaps/android-maps-compose#346]
#### Environment details

1. street view compose
2. android api 27 emulator
3. latest commit as of today

#### Steps to reproduce

  1. clone android-maps-compose
  2. edit BasicMapActivity.kt to use the [Readme's location](https://github.com/googlemaps/android-maps-compose/blob/main/README.md#street-view):
  
```
- val singapore = LatLng(1.3588227, 103.8742114)  
+ val singapore = LatLng(1.35, 103.87)
```

3. Run
Crashes, same as #265 

#### Stack trace
```
FATAL EXCEPTION: main
                                                                                         Process: com.google.maps.android.compose, PID: 10410
                                                                                         com.google.maps.api.android.lib6.common.apiexception.f: java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter it
                                                                                         	at com.google.maps.api.android.lib6.streetview.camera.i.g(:com.google.android.gms.dynamite_mapsdynamite@230914022@23.09.14 (040700-0):23)
                                                                                         	at com.google.maps.api.android.lib6.streetview.camera.e.run(:com.google.android.gms.dynamite_mapsdynamite@230914022@23.09.14 (040700-0):0)
                                                                                         	at android.os.Handler.handleCallback(Handler.java:790)
                                                                                         	at android.os.Handler.dispatchMessage(Handler.java:99)
                                                                                         	at android.os.Looper.loop(Looper.java:164)
                                                                                         	at android.app.ActivityThread.main(ActivityThread.java:6494)
                                                                                         	at java.lang.reflect.Method.invoke(Native Method)
                                                                                         	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
                                                                                         	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
                                                                                         Caused by: java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter it
                                                                                         	at com.google.maps.android.compose.streetview.StreetViewPanoramaPropertiesNode.onAttached$lambda$3(Unknown Source:7)
                                                                                         	at com.google.maps.android.compose.streetview.StreetViewPanoramaPropertiesNode.$r8$lambda$u0P0-MFqw789nPtol3qohzgcP-I(Unknown Source:0)
                                                                                         	at com.google.maps.android.compose.streetview.StreetViewPanoramaPropertiesNode$$ExternalSyntheticLambda3.onStreetViewPanoramaChange(Unknown Source:2)
                                                                                         	at com.google.android.gms.maps.zzai.zzb(com.google.android.gms:play-services-maps@@18.0.0:1)
                                                                                         	at com.google.android.gms.maps.internal.zzbk.zza(com.google.android.gms:play-services-maps@@18.0.0:2)
                                                                                         	at com.google.android.gms.internal.maps.zzb.onTransact(com.google.android.gms:play-services-maps@@18.0.0:3)
                                                                                         	at android.os.Binder.transact(Binder.java:627)
                                                                                         	at fh.c(:com.google.android.gms.dynamite_mapsdynamite@230914022@23.09.14 (040700-0):2)
                                                                                         	at com.google.maps.api.android.lib6.streetview.camera.i.g(:com.google.android.gms.dynamite_mapsdynamite@230914022@23.09.14 (040700-0):22)
                                                                                         	at com.google.maps.api.android.lib6.streetview.camera.e.run(:com.google.android.gms.dynamite_mapsdynamite@230914022@23.09.14 (040700-0):0) 
                                                                                         	at android.os.Handler.handleCallback(Handler.java:790) 
                                                                                         	at android.os.Handler.dispatchMessage(Handler.java:99) 
                                                                                         	at android.os.Looper.loop(Looper.java:164) 
                                                                                         	at android.app.ActivityThread.main(ActivityThread.java:6494) 
                                                                                         	at java.lang.reflect.Method.invoke(Native Method) 
                                                                                         	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438) 
                                                                                         	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807) 
```

Following these steps will guarantee the quickest resolution possible.

Thanks!

**Comment by wangela** [comment]
If you would like to upvote the priority of this issue, please comment below or react with :+1: so we can see what is popular when we triage.
--------

@lawm Thank you for opening this issue. 🙏
Please check out these other resources that might help you get to a resolution in the meantime:

- [Check the issue tracker](https://developers.google.com/maps/support/#issue_tracker) - bugs and feature requests for Google Maps Platform APIs and SDKs
- [Open a support case](https://developers.google.com/maps/support#contact-maps-support) - Get 1:1 support in Cloud Console.
- [Discord](https://discord.gg/fHTDmZe9En) - chat with other developers
- [StackOverflow](http://stackoverflow.com/questions/tagged/google-maps) - use the `google-maps` tag


_This is an automated message, feel free to ignore._

**Comment by wangela** [comment]
Fixed by #345
